### PR TITLE
Fix: Update formatter html to support new severities

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

**Comment by sarvaje** [comment]
Until we have the new design, I think this is all we need to change.

File: packages/formatter-html/src/result.ts [hunk 1]
```diff
@@ -174,15 +171,12 @@ export class CategoryResult {
         let hint = this.getHintByName(hintId);
 
         if (!hint) {
-            // All the problems have to have the same severity, so we just need to calculate the status once.
-            const status = problem.severity === Severity.error ? 'error' : 'warning';
-
-            hint = new HintResult(hintId, status, this.url, this.isScanner);
+            hint = new HintResult(hintId, this.url, this.isScanner);
```
Comment (molant) [inline packages/formatter-html/src/result.ts]: How are we indicating the severity of each report? Can you paste a screenshot?

Comment (sarvaje) [reply packages/formatter-html/src/result.ts]: We don't do that in this formatter

**Comment by molant** [comment]
We really need to get #3179 done 😓

**Comment by sarvaje** [comment]
> We really need to get #3179 done 😓

Yep

**Pull request merged**